### PR TITLE
Populate Nautobot Lab with Sandbox Data

### LINKED ISSUE #8: Pre-load data set and superuser
- [x] Pre-load a super user to make consuming the lab easier: https://github.com/nautobot/nautobot/blob/develop/development/docker-entrypoint.sh
- [x] Pre-load data into Nautobot to make kicking the tires more interesting.

### PR DESCRIPTION
**Pull request by jtdub**
Fixes #8 

Loads a set of mock data from the `demo.nautobot.com` dataset.

### CODE REVIEW

File: pb_nautobot_install.yml [hunk 1]
```diff
@@ -79,8 +79,15 @@
...
-      ansible.builtin.command: "/usr/local/bin/nautobot-server migrate"
+    # - name: "RUN MIGRATE"
+    #  ansible.builtin.command: "/usr/local/bin/nautobot-server migrate"
+
+    - name: "LOAD NAUTOBOT DATA INTO DATABASE"
+      ansible.builtin.command: "/opt/nautobot/templates/loaddata.sh"
+      environment:
+        PGUSER: "{{ lookup('env', 'DB_USER') }}"
+        PGPASSWORD: "{{ lookup('env', 'DB_PASSWORD') }}"
+        PGDATABASE: "{{ lookup('env', 'DB_NAME') }}"
```
Comment (jvanderaa) [inline pb_nautobot_install.yml]: Should probably add a `when: "{{ lookup('env', 'LOAD_DUMP_DATA') }}" to this, adding as a default yes, but a way to exclude it.

Comment (nniehoff) [reply pb_nautobot_install.yml]: I think the point of the lab is to have users execute `docker run...` and not have to build the container.  In which case I don't really see the point of having options via environment variables that only impact the container build.

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Need to add the env var with default to no for adding superuser creds.

**Comment by jtdub** [comment]
@jvanderaa -- Setting an environment variable will only affect the `docker build` process. It wouldn't affect `docker run`

**Comment by jvanderaa** [comment]
So then moving it to an entrypoint creation?

File: pb_nautobot_install.yml (hunk 1 above)
Comment (jvanderaa) [reply pb_nautobot_install.yml]: That's a good point. So the conditionals need to be moved to another place. Or are we looking at multiple container images?

- AIO empty
- AIO Preloaded
- AIO All the Plugins loaded
- AIO Onboarding only

??

**Pull request merged**